### DISCUSSION TITLE
How to stop/cancel an ongoing token generation?

### DISCUSSION THREAD

**Discussion by DanielProkhorov** [amithkoujalgi/ollama4j#55] [Q&A]

**Comment by amithkoujalgi** [comment]
There is no such API available from Ollama at the moment. If Ollama adds this API, we can add support for this in Ollama4j.

For your reference: https://github.com/ollama/ollama/blob/main/docs/api.md 

Thanks!

**Comment by reckart** [comment]
Assuming you use streaming mode, you could probably just close the ongoing connection.
